besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1760,6 +1762,11 @@ private void startPlugins() {
                 besuController.getProtocolContext().getWorldStateArchive()),
             besuController.getProtocolSchedule()));
 
+    besuPluginContext.addService(
+        WorldViewService.class,
+        new WorldViewServiceImpl(
+            besuController.getProtocolContext().getWorldStateArchive().getMutable()));
```

Comment:
It is quite risky to expose the 'root' worldstate from bonsai.  Could we instead expose BlockchainQueries or a similar functional interface like getAndMapWorldState() that has some safety to prevent accidental mutation?  

https://github.com/hyperledger/besu/blob/main/ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/query/BlockchainQueries.java#L904-L915

I believe the TracerService plugin already has blockchainQueries, we can keep with that pattern.